client/src/Utils/Validations/Validator.test.ts

```diff
@@ -0,0 +1,86 @@
+import Validator from './Validator';
+
+describe('Validation Tests', () => {
+    describe('PhoneValidation Tests', () => {
+
+        describe('CellPhone Numbers: ', () => {
+            it('Should be true (correct start & correct number of digits)', () => {
+                expect(Validator.phoneValidation('050-1234561')).toBeTruthy();
+            });
+
+            it('Should be true (correct start & correct number of digits)', () => {
+                expect(Validator.phoneValidation('051-1234561')).toBeTruthy();
+            });
+
+            it('Should be true (correct start & correct number of digits)', () => {
+                expect(Validator.phoneValidation('052-1234561')).toBeTruthy();
+            });
+
+            it('Should be true (correct start & correct number of digits)', () => {
+                expect(Validator.phoneValidation('053-1234561')).toBeTruthy();
+            });
+
+            it('Should be true (correct start & correct number of digits)', () => {
+                expect(Validator.phoneValidation('054-1234561')).toBeTruthy();
+            });
+
+            it('Should be true (correct start & correct number of digits)', () => {
+                expect(Validator.phoneValidation('055-1234561')).toBeTruthy();
+            });
```

Comment:
what's the difference between them? if there is one, maybe provide an explanation in the test description

Reply:
הקידומת אין איך להסביר

Reply:
beseder